bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerEntry.java

```diff
@@ -23,41 +23,37 @@
 import com.google.common.base.Preconditions;
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.ByteBufInputStream;
-
 import java.io.InputStream;
+import org.apache.bookkeeper.client.impl.LedgerEntryImpl;
 import org.apache.bookkeeper.conf.ClientConfiguration;
 
 /**
  * Ledger entry. Its a simple tuple containing the ledger id, the entry-id, and
  * the entry content.
  *
+ * @Deprecated since 4.6.0, in favor of using {@link org.apache.bookkeeper.client.impl.LedgerEntryImpl}.
  */
-public class LedgerEntry
-    implements org.apache.bookkeeper.client.api.LedgerEntry {
+@Deprecated
```

Comment:
don't add `@Deprecated` at this release.

Reply:
get it.